string dateTime
float64[3] position 
float64[3] velocity
float32[4] temp
float32[4] relativeHum
float32 lightInt
float64 par
float32 CO2level
